proto/src/determined/api/v1/experiment.proto

```diff
@@ -631,6 +631,8 @@ enum MetricType {
   METRIC_TYPE_TRAINING = 1;
   // For metrics emitted during validation.
   METRIC_TYPE_VALIDATION = 2;
+  // For metrics emitted during profiling.
+  METRIC_TYPE_PROFILING = 3;
```

Comment:
https://github.com/determined-ai/determined/blob/main/master/pkg/model/metrics.go#L41
https://github.com/determined-ai/determined/blob/main/master/pkg/protoutils/protoconverter/converter.go#L169

These functions may need to be updated to work with the new type.

Reply:
the READ part of getting profiling metrics from generic metrics isn't implemented in this feature, only the write. so i think it's appropriate for the time being that those functions return "unspecified" or "unknown", because at this point in time, profiling metrics should not be known to those APIs.